docs/README-develop-types-providers.md

```diff
@@ -24,7 +24,7 @@
 
 This document is a guide for writing new Puppet resource types and providers for Cisco NX-OS.
 
-There are multiple components involved when creating new resources. This document focuses on the type and provider files:
+There are multiple components involved when creating new resources, it would be suggested to review [`custom resources`](https://puppet.com/docs/puppet/6.0/custom_resources.html) as it outlines the various ways a custom type and provider can be written for Puppet. This document focuses on the legacy type and provider files:
```

Comment:
Add a reference to `pdk new provider` as THE way to get started. This now also needs to capture the `remote_resource` workaround like https://github.com/puppetlabs/netdev_stdlib/blob/088b60f83fc7051a1c96fb57c5e8c6816f683794/lib/puppet/type/network_interface.rb#L69